File: inputs/c7/codegolf_8.cpp

#include <iostream>
#include <fstream>
#include <sstream>
#include <string.h>
#include <iomanip>

CODEGOLF_METHOD_PLACEHOLDER


int main(int argc, char* argv[])
{
    std::string building = "1:0|1:1\n1:3|5:5|2:0\n\n3:2|5:0\n1:1|2:2";
                    int capacity = 10;
                    
    f(building, capacity);

    return 0;
}

File: inputs/c7/codegolf_8.go

package main

import (
    "fmt"
    "strings"
    "bytes"
    "io/ioutil"
    "strconv"
    "runtime"
    "regexp"
)

func main() {
    var building string = "1:0|1:1\n1:3|5:5|2:0\n\n3:2|5:0\n1:1|2:2"
                    var capacity int = 10
                    
    f(building, capacity)
}

CODEGOLF_METHOD_PLACEHOLDER


func dummyfuncforincludes(){
    dat, err := ioutil.ReadFile("pear")

    if err != nil {
        panic(err)
    }

    fmt.Println(string(dat))

    var buffer bytes.Buffer
    fmt.Println(buffer.String())

    b, err := strconv.ParseBool("true")
    if err != nil {
        panic(err)
    }

    fmt.Println(strconv.FormatBool(b))

    r := []string{}
    strings.Join(r, "")

    fmt.Println(runtime.Compiler, runtime.GOARCH, runtime.GOOS)
    fmt.Println(strconv.IntSize)

    regexp.QuoteMeta("a")
}


